docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/svpc_liver-cancer-causes-risk-factors-and-prevention.content.yml

```diff
@@ -15,7 +15,7 @@
   field_pdq_audience:
     value: Patients
   field_pdq_summary_type:
-    value: Causes, risk factors, and prevention
+    value: Genetics
```

Comment:
Why and how did this change??

Reply:
The yaml was updated because the previous value was an incorrect value and it is now being used as part of the meta tag for templates so it had to be changed to a correct value.